src/payment/PaymentPage.jsx

```diff
@@ -27,11 +27,15 @@ class PaymentPage extends React.Component {
     const {
       experimentVariables: {
         isPaymentVisualExperiment = false,
+        isNumEnrolledExperiment = false,
```

Comment:
I would personally use just or the other of these experiment variables, assuming all the changes are one experiment. Not a big deal either way

Reply:
Thought it would be better to keep it separate in case there is another experiment that requires to turn on only one of them